tests/test_layout_objects.py

```diff
@@ -151,6 +153,21 @@ def test_remove_labels():
     assert "<label" not in html
 
 
+@pytest.mark.parametrize(
```

Comment:
I think this is great, so many tests can be covered here without lots of repetition. (Although I struggle a little reading the parameterize function)